<launch>
  <arg name="rate" default="10" />
  <arg name="namespace" default="$(env UAV_NAMESPACE)" />
  <arg name="wp_radius" default="0.1" />
  <arg name="tracker_input_type" default="pose" />

  <group ns="$(arg namespace)">
    <node name="path_to_tracker" pkg="path_planning_optimization" type="path_to_tracker.py" output="screen">
      <param name="rate" value="20" />
      <param name="wp_radius" value="0.3" />
      <param name="tracker_input_type" value='trajectory' />
    </node>
  </group>
</launch>
